useful-scripts/misc-test-utils/src/modules/subject/uploadSubject.ts

```diff
@@ -40,24 +39,23 @@ const uploadSingleSubject = async (releaseId: string) => {
     releaseId,
   );
 
-  if (!importStatus) {
-    console.log(
-      'No importStatus just yet, waiting 4 seconds before polling again',
-    );
-    await sleep(4000);
-  }
-
-  while (importStatus !== 'COMPLETE') {
-    console.log(chalk.blue('importStatus', chalk.green(importStatus)));
-    // eslint-disable-next-line no-await-in-loop
-    await sleep(1000);
-
-    // eslint-disable-next-line no-await-in-loop
+  while (importStatus !== 'QUEUED') {
     importStatus = await subjectService.getSubjectProgress(
       releaseId,
       subjectId as string,
     );
   }
+
+  if (!fast) {
+    while (importStatus !== 'COMPLETE') {
+      console.log(chalk.blue('importStatus', chalk.green(importStatus)));
+      importStatus = await subjectService.getSubjectProgress(
+        releaseId,
+        subjectId as string,
+      );
+      await sleep(3000);
```

Comment:
Could make this 3000 a constant, or a variable that can be supplied in .env?  Not to worry as part of this PR though, not important.

Reply:
Good point. I've moved this to an env var